include/render/gles2.h

```diff
@@ -72,6 +72,7 @@ struct wlr_gles2_texture {
 	enum wlr_gles2_texture_type type;
 	int width, height;
 	bool has_alpha;
+	uint32_t wl_format; // used to interpret upload data
```

Comment:
This can be `enum wl_shm_format`

Reply:
ok, will change it in wlr_gles2_pixel_format as well then.